mount.yazi/main.lua

```diff
@@ -179,8 +187,14 @@ function M.obtain()
 			main, sub = p.src:match("^(/dev/disk%d+)(.+)$")
 		elseif p.src:find("/dev/nvme", 1, true) == 1 then -- /dev/nvme0n1p1
 			main, sub = p.src:match("^(/dev/nvme%d+n%d+)(p%d+)$")
+			if sub then
+				p.fstype = p.fstype or get_fstype(p.src)
```

Comment:
Please merge multiple `lsblk` into one, i.e. you can pass several partitions to `lsblk` like this:

```sh
lsblk -n -o fstype /dev/sda1 /dev/sda2
```